pkg/cluster/cluster_config.go

```diff
@@ -257,5 +265,47 @@ func getClusterInfo(ctx context.Context, cli client.Client) (ClusterInfo, error)
 	}
 	c.Version = ocpVersion
 
+	// Check for FIPs
+	if fipsEnabled, err := isFipsEnabled(ctx, cli); err == nil {
+		c.FipsEnabled = fipsEnabled
+	} else {
+		logf.FromContext(ctx).Info("could not determine FIPS status, defaulting to false", "error", err)
+	}
+
 	return c, nil
 }
+
+func isFipsEnabled(ctx context.Context, cli client.Client) (bool, error) {
+	// Check the install-config for the fips flag and it's value
+	// https://access.redhat.com/solutions/6525331
+	cm := &corev1.ConfigMap{}
+	namespacedName := types.NamespacedName{
+		Name:      "cluster-config-v1",
+		Namespace: "kube-system",
+	}
+
+	if err := cli.Get(ctx, namespacedName, cm); err != nil {
+		return false, err
+	}
+
+	if _, ok := cm.Data["install-config"]; ok {
```

Comment:
I think you can remove this if block entirely, `cm.Data["install-config"]` would return an empty string if the key is not present, so it should be covered by the `if` below